bundles/org.openhab.binding.lcn/src/main/java/org/openhab/binding/lcn/internal/LcnBindingConstants.java

```diff
@@ -25,6 +25,8 @@
 public class LcnBindingConstants {
     /** The scope name of this binding */
     public static final String BINDING_ID = "lcn";
+    /** Name of the module's serial number property */
+    public static final String SERIAL_NUMBER_PROPERTY = "serialNumber";
```

Comment:
Looks as if you are not yet aware of `Thing.PROPERTY_SERIAL_NUMBER` which should definitely be used here!

Reply:
Indeed, great! Fixed.